helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -486,187 +577,273 @@ public void updateDeploymentGroupHosts(final String groupName, final List<String
     }
   }
 
-  @Override
-  public void rollingUpdate(final DeploymentGroup deploymentGroup,
-                            final JobId jobId,
-                            final RolloutOptions options)
-      throws DeploymentGroupDoesNotExistException, JobDoesNotExistException {
-    checkNotNull(deploymentGroup, "deploymentGroup");
+  private void removeRollingOperation(final ZooKeeperClient client,
+                                      final List<ZooKeeperOperation> operations,
+                                      final String rollingOpId) {
+    try {
+      try {
+        final RollingOperationStatus status = getRollingOperationStatus(client, rollingOpId);
+        if (status.getState() == ROLLING_OUT) {
+          throw new HeliosRuntimeException(
+              "Please wait for rolling-operation " + rollingOpId + " to finish.");
+        }
+      } catch (RollingOperationDoesNotExistException e) {
+        return;
+      }
+      client.ensurePath(Paths.configRollingOps());
+      client.ensurePath(Paths.statusRollingOps());
```

Comment:
Shouldn't we also remove `Paths.statusRollingOpsTasks`? And shouldn't we use the [same code @gimaker had here](https://github.com/spotify/helios/pull/903/files#diff-8d500c531ffb1f7836e9d5b60585a002L408) to ensure consistency?